config/initializers/act_as_attachable.rb

```diff
@@ -0,0 +1,3 @@
+require 'act_as_attachable'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.